source/manuals/logit-io.html.md.erb

```diff
@@ -49,11 +49,21 @@ Logstash speaks many different protocols and these are all assigned to sequentia
 
 TCP ports are assigned on a per-stack basis at stack creation time and will be different per stack.
 
-## Shipping logs via Amazon S3
+## Send logs from virtual machines
+
+*what do we need here*
```

Comment:
Here we can have something about deciding between running Logstash or using the Logstash provided by Logit.